src/pramod/StringLowerCaseAssignment19.java

```diff
@@ -0,0 +1,26 @@
+package pramod;
+
+public class StringLowerCaseAssignment19 {
+	
+	int stringLowerCaseNumber(String str)
+	{  
+		int count=0;
+		str = str.toLowerCase();
+		for(int i=0;i<str.length();i++)
+		{
+			if(str.charAt(i)=='t')
```

Comment:
Don't make this hardcoded value, make it generic so that you can use this code for other values too